.devcontainer/entrypoint.sh

```diff
@@ -63,4 +56,5 @@ if [ "$1" = "--server" ]; then
 fi
 
 cd /workspace/comfystream
-/bin/bash
\ No newline at end of file
+
+exec "$@"
```

Comment:
@eliteprox do you see issues with forwarding the remaining arguments? Now I can use the container directly in interactive mode.

Reply:
Verified it's working correctly!